### DISCUSSION TITLE
Any thoughts on Kompute Bindings

### DISCUSSION THREAD

**Discussion by davidbruce** [LWJGL/lwjgl3#706] [Ideas]
Hey I was wondering if anyone has considered adding Kompute to LWJGL. 
https://github.com/KomputeProject/kompute

Any good resources I should read through if I wanted to try adding them myself?

**Comment by SWinxy** [comment]
LWJGL can't do C++ bindings yet, and it doesn't seem like Kompute has a C API.

**Reply by davidbruce** [reply]
Ah gotcha I might have to checkout JavaCPP then and see if some stand alone bindings would be possible.  Thanks for letting me know!
